modules/UI/videolayout/RemoteVideo.js

```diff
@@ -182,6 +205,68 @@ RemoteVideo.prototype._generatePopupContent = function () {
     return popupmenuElement;
 };
 
+/**
+ * Sets the remote control supported value and initializes or updates the menu
+ * depending on the remote control is supported or not.
+ * @param {boolean} isSupported
+ */
+RemoteVideo.prototype.setRemoteControlSupport = function(isSupported = false) {
```

Comment:
It would be useful to have some info logged about participant's support for remote control (for debugging purpose).